account-kit/signer/src/base.ts

```diff
@@ -715,6 +724,9 @@ export abstract class BaseAlchemySigner<TClient extends BaseSignerClient>
         authenticatingType: "email",
       });
 
+      // fire new user event
+      this.emitNewUserEvent(params.isNewUser!);
```

Comment:
this isn't guaranteed to be passed in, so I would make it optional and handle it appropriately in the method